src/lib/services/party.ts

```diff
@@ -41,4 +41,26 @@ export default class Party {
       },
     });
   }
+
+  static async getPartyWithUsers({ partyHash }: GetParams) {
+    return dbClient.parties.findFirst({
+      where: {
+        hash: partyHash,
+      },
+      include: {
+        users: true,
+      },
+    });
+  }
```

Comment:
```suggestion
  static async getPartyWithUsers({ partyHash }: GetParams) {
    return dbClient.parties.findFirst({
      where: {
        hash: partyHash,
      },
      include: {
        users: true,
      },
    });
  }
```

```
  static async searchUsers({ partyHash }: GetParams) {
    return dbClient.parties.findFirst({
      where: {
        hash: partyHash,
      },
      select: {
        users: true,
      },
    });
  }
```

This should exclusively select users which should eliminate the need for the below method